.github/workflows/deploy.yml

```
@@ -52,6 +52,7 @@ jobs:
             --no-progress
             --base "$(pwd)/build"
             --accept 100..399,401,403,429
+            --exclude 'https://conda-forge/status/'
```

To fix link check, is this what we want? Or should this be changed somehow?

---

```suggestion
            --exclude '^.*/status/migration/.*$'
```

I think this should work. Let's see.